### DISCUSSION TITLE
Add new pre-built scenario ride names from RCT Classic when importing RCT1 AA/LL scenarios

### DISCUSSION THREAD

**Discussion by ZeeMaji** [OpenRCT2/OpenRCT2#16269] [Feature requests] [answered]
RCT Classic's scenarios provide proper names for many of the pre-built rides in AA/LL's scenarios that previously lacked proper names.
Most of these are for LL scenarios but there's a couple for AA scenarios, I think it would be a nice touch to implement these new ride names when importing the original rct1 scenarios into openrct2.
Tiny Towers' coasters in rctc technically have names but they are all named "Roller Coaster #" as though they were directly imported from rct1 so i have excluded them. 
I also used the OpenRCT2 names for the ride types (eg: classic mini coaster instead of steel mini coaster)
Asterisk symbol means it is a different ride type in rct classic but it is otherwise the same ride in the scenario's rct1 counterpart.

Added Attractions/Corkscrew Follies:

Future World
Suspended Monorail > Lunavator

Fruit Farm
Miniature Railroad > Orchard Railway

Fiasco Forest
Boat Hire > Boat Jam

Loopy Landscapes:

Iceberg Isles
Bobsleigh Coaster > Iceberg Bobs
Mini Golf > Iceberg Golf
Chairlift > Iceberg Chairlift

Volcania
Wooden Coaster > Volcanic Wooden Roller Coaster
Steel Wild Mouse > Volcanic Wild Mouse
Suspended Swinging Coaster > Volcanic Suspended Coaster
Looping Coaster > Volcanic Looping Roller Coaster
Compact Inverted Coaster > Volcanic Inverted Coaster

Vertigo Views
Hypercoaster > Vertigulator

Paradise/Big Pier 2
Classic Mini Coaster > Beach Buggies*

Dragon's Cove
Suspended Swinging Coaster > Dragon Planes
Vertical Drop Coaster > Dragon's Drop
Looping Coaster > Dragon Loops
Compact Inverted Coaster > Dangling Dragons
Bobsleigh Coaster > Dragon Sleigher

Good Knight Park
Reverser Coaster 1 > Knight Shift 1
Reverser Coaster 2 > Knight Shift 2

Wacky Warren
Classic Mini Coaster > Wiggly Worm*
Inverted Hairpin Coaster > Rat Flyer

Dusty Desert
Inverted Coaster > Dusty Dangler
Twister Coaster > Dusty Twister
Lay-down Coaster > Dusty Flight
Wooden Coaster > Dusty Rattler
Vertical Drop Coaster > Dusty Drop

Woodworm Park
Miniature Railroad > Woodworm Railroad
Boat Hire > Woodworm Boats

Sunny Swamps
Water Coaster > Swamp Boats
River Rafts > Swamp Rafts

Thunder Rocks
Vertical Drop Coaster > Thunder Drops

Southern Sands
Wooden Coaster > Desert Diver
Compact Inverted Coaster > Sandstorm
Monorail > Burrower

Nevermore Park
Classic Mini Coaster > Bugulator*

**Comment by HtotheTML** [comment]
Were any of the shops/stalls renamed?

**Reply by ZeeMaji** [reply]
I'm not sure, it's been so long since i checked these.

From what i can remember: no, they weren't. But i'm not certain on that.

**Reply by HtotheTML** [reply]
Just looked through them all, it appears none of them were changed.

**Reply by tupaschoal** [reply]
FWIW it is probably easy to write a plugin that dumps all ride names upon loading a scenario then it might also be possible to write one that loops through all scenarios.

Since you checked them all though, that might not be needed anymore :D

**Comment by ZeeMaji** [comment, marked as answer]
Implemented by #27017
